natsjobs/config.go

```diff
@@ -5,6 +5,8 @@ import (
 )
 
 const (
+	NatsSubjectHeaderKey string = "x-nats-subject"
```

Comment:
Should be lowercase first letter. This is not imported constant.

Reply:
And you don't need to use `Nats`. Since this is the `nats` package, `subjectHeaderKey` would be enough.

Reply:
Fixed :)